api/proto/config/baseosconfig.proto

```diff
@@ -28,8 +28,22 @@ message BaseOSConfig {
   // exist for backward compatibility.
   // Drives will be deprecated in the future.
   repeated Drive drives = 3;
+
+  // OBSOLETE - Device must ignore EdgeDevConfig.baseos.activate.
```

Comment:
No, all of BaseOSConfig should be marked as Deprecated not just a few fields.

Reply:
So just start over with the new BaseOs Structure? But that is not possible yet without moving EVE Images to Volumes - right? For example, as it exists currently:

repeated Drive drives = 3; --> Still needed

So thought it is better to just evolve the current structure rather than create a new one. The fields can be obsoleted when we are ready.